attls/Jenkinsfile

```diff
@@ -0,0 +1,88 @@
+def serverCredentials = []
+def ARTIFACTORY_CREDENTIALS_ID = 'zowe.jfrog.io'
+Map TEST_SERVERS = [
+        'marist'  : [
+                ansible_host     : 'marist-1',
+                ssh_hostport     : 'ssh-marist-server-zzow01-hostport',
+                ssh_userpass     : 'ssh-marist-server-zzow01',
+                node_home_pattern: '/ZOWE/node/node-{NODE_VERSION}-os390-s390x'
```

Comment:
Node is ignored on the other two servers or use the same?

Reply:
I took the map from zowe install packaging so I am not sure what is the actual situation